Tools/CISupport/ews-build/steps.py

```diff
@@ -1516,7 +1516,7 @@ def start(self):
             self.finished(SUCCESS)
             return None
 
-        self.setProperty('reviewer_full_name', self.full_name_from_email(reviewer))
+        self.setProperty('reviewer_full_names', [self.full_name_from_email(reviewer)])
```

Comment:
Who is setting the 'reviewer' property? Seems like it is being set only for patch workflow, and currently we can't have multiple reviewers in that. It might be better to have this patch after or along-with the patch when we actually support having multiple reviewers in 'reviewer' property (for git workflow), but I am ok landing it before-hand as well if you prefer so.

Reply:
Also, reviewer_full_names => reviewers_full_names